### PR TITLE
Planet Four: convert retirement page to author page

### PR DESCRIPTION
**Pull request by lcjohnso**
Building from @wgranger's #49, I'm taking things a step farther by converting the retirement page into an author page.  Specifically, I swap the `/author` content for `/index` content and strip out the original `/index` and `/results` sections.

The idea is to deploy this as a static site on S3 at a new `authors.planetfour.org` domain, but where the published URL of `planetfour.org/authors` will also redirect here.  The redirect is addressed in https://github.com/zooniverse/static/issues/117.

This is a bit of a hack and need testing and review before merge and deploy.  I think it's basically working, but I'm no expert, so will defer to @wgranger and others!  Also: I'm submitting this as an alternate PR because I wasn't sure which case we should adopt and didn't want to layer commits that might need to be reverted.  Might still decide this isn't the way to go.

For clean-up, the current retirement page at `planetfour.org` can also be removed from S3 zooniverse-static.

### CODE REVIEW

File: PlanetFourAuthors/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Planet Four: Archive</title>
+        <meta name="description" content="">
```
Comment (wgranger) [inline PlanetFourAuthors/index.html]: Just one comment about these two lines. 

**Line 9:** Are we still happy with the title being "Planet Four: Archive?" We could also change it to "Planet Four: Authors," which may be more specific.

**Line 10:** I believe this line is used for search engine optimization, so we should probably give it more content than an empty string. Usually we use "Archived Zooniverse Project: Planet Four," but we could even be a bit more detailed and use "Archived Zooniverse Project: Planet Four contributing authors," or something similar.

**Review by wgranger — APPROVED** [review]
Just one comment that we should probably address, but it isn't at all blocking. Also, good catch in changing the GA ID on this PR, which is an easy thing to overlook.

File: PlanetFourAuthors/index.html (hunk 1 above)
Comment (eatyourgreens) [reply PlanetFourAuthors/index.html]: A couple of comments on the HTML, also not blocking.

- the `lang` attribute on the `html` tag can't be an empty string. `lang="en"` (or `lang="en-US"`) will tell user agents the page is in English. Search engines use this attribute when indexing pages. Screen-readers use it too.
- the conditional comments that add extra CSS classes for older versions of IE probably aren't needed any more.

Comment (lcjohnso) [reply PlanetFourAuthors/index.html]: Made changes to both of the items Will pointed out -- thanks!

Comment (lcjohnso) [reply PlanetFourAuthors/index.html]: Added lang="en" to html tag as Jim suggested; kept conditionals as is (no harm no foul?).

**Pull request merged**